lesson14/style.css

```diff
@@ -7,17 +7,19 @@ a > img {
   width: 18px;
 }
 
-#js-btn-fetch {
+#js-btn-fetch,
+#js-btn-showModal {
   padding: 15px 25px;
   background-color: beige;
-  margin-left: 25px;
+  /* margin-left: 25px; */
   border-radius: 8px;
   outline: none;
   border: 1px solid #aaa;
   cursor: pointer;
 }
 
-#js-btn-fetch:hover {
+#js-btn-fetch:hover,
+#js-btn-showModal:hover {
```

Comment:
same [this comment](https://github.com/hippopo-moo/js-handson-work/pull/15/files#r661269506)

Reply:
Thank you.
I fixed it.